Test/dafny0/formatting1-3.dfy

```diff
@@ -0,0 +1,2 @@
+// RUN: %diff "%s" "%s"
```

Comment:
What's the purpose of this line? (and same for `formattinginclude.dfy`)

Reply:
> What's the purpose of this line? (and same for formattinginclude.dfy)

I got an error from the test engine saying that it could not find a lit command in this Dafny file.
Because I don't know how to skip files from being tested at this point, I added a dummy test.

Reply:
I found that lit tests were automatically triggered on any Dafny file, and were failing on these files because they don't contain lit commands. So my workaround is to add a dummy lit command that will just always pass, but there is nothing to test. I couldn't find lit documentation other than our own usage of lit.

Reply:
Several of our test files that don't have commands contain `// RUN: echo ''` to be mostly ignored by lit.

Reply:
That works as well! Would you prefer that approach?

Reply:
Maybe `// RUN: echo 'lit should ignore this file'` to be super clear